### ISSUE TITLE
Appveyor tests failing because ¯\_(ツ)_/¯

### ISSUE THREAD

**Issue by qtothec** [Pyomo/pyomo#542]
PR #539 fixed Appveyor 3.6 (#538) but broke the 3.5 tests somehow. They are now failing with the message:
```
...
test_call (pyomo.core.tests.unit.test_component_piecewise.Test_piecewise_nd) ...
Intel MKL FATAL ERROR: Cannot load mkl_intel_thread.dll.
```

**Comment by qtothec** [comment]
This is now an issue with the 2.7 and 3.6 tests as well.

**Comment by qtothec** [comment]
This was resolved in #577
